modules/management-console-agent/src/main/java/org/apache/ignite/agent/dto/action/RequestDeserializer.java

```diff
@@ -73,7 +77,7 @@ public RequestDeserializer() {
             }
         }
         catch (Exception e) {
-            req = new InvalidRequest().setCause(e).setId(id).setAction(act);
+            req = new InvalidRequest().setCause(new IllegalArgumentException(e)).setId(id).setAction(act);
```

Comment:
Why did you wrap `e` with IllegalArgumentException ?

Reply:
Here we catch exceptions during parsing, and I think it is worth simplifying such exceptions in IllegalArgumentException to simplify further processing